app/decorators/user_decorator.rb

```diff
@@ -116,6 +116,14 @@ def devices?
     !recent_devices.empty?
   end
 
+  def last_signed_in_at
+    user.devices.order(last_used_at: :desc).first&.last_used_at
+  end
+
```

Comment:
```suggestion
```

It looks like this isn't used?

Reply:
Yeah, this isn't used. It can be removed. I originally included it because it seemed odd to have a method for `second_last_signed_in_at` but no method for `last_signed_in_at`.